container/shim/src/utils/system.js

```diff
@@ -10,7 +10,7 @@ const exec = promisify(CpExec);
 
 const debug = Debug.extend("system");
 
-const meminfoKBToGB = (bytes) => bytes / 1_000_000;
+const meminfoKBToGB = (bytes) => (bytes / 1_000_000).toFixed(0);
```

Comment:
Use powers of 2 instead of 10? e.g. `1048576` (1024 ** 2) instead of `1_000_000`

Reply:
Nope, reports incorrect info. Base 10 reports correctly for both memory and disk

Reply:
4027844 kB -> 4GB using base 10, 3.84 GB using base 2.